lib/modules/announcements.js

```diff
@@ -2,7 +2,7 @@
 import _ from 'lodash';
 import { $ } from '../vendor';
 import * as Metadata from '../core/metadata';
-import { DAY, HOUR, isCurrentSubreddit, CreateElement } from '../utils';
+import { DAY, HOUR, isCurrentSubreddit, CreateElement, BrowserDetect } from '../utils';
```

Comment:
Iinstead of BrowserDetect.browser, you could probably use process.env.BUILD_TARGET